### ISSUE TITLE
For RN 0.56.0: Android SDK Build Tools revision (23.0.1) is too low

### ISSUE THREAD

**Issue by sunweiyang** [opentok/opentok-react-native#145]
On RN 0.56.0, when trying to build Android app, I see the following error output:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugApk'.
   > A problem occurred configuring project ':opentok-react-native'.
      > The SDK Build Tools revision (23.0.1) is too low for project ':opentok-react-native'. Minimum required is 25.0.0
```

As of RN 0.56.0, the Android SDK versions used have been changed to the following:
```
    buildToolsVersion = "26.0.3"
    minSdkVersion = 16
    compileSdkVersion = 26
    targetSdkVersion = 26
    supportLibVersion = "26.1.0"
```

**Comment by msach22** [comment]
@sunweiyang Thanks for letting us know! If you're up for it, please send a PR for the following file: https://github.com/opentok/opentok-react-native/blob/master/android/build.gradle

If not, I can get to it this week 😄

**Comment by sunweiyang** [comment]
Created PR #148 -- reviews welcome.

**Comment by msach22** [comment]
Thanks for the PR @sunweiyang! This has been released with `v0.8.4`
